Tools/YamlCreate.ps1

```diff
@@ -110,7 +110,7 @@ Function Read-PreviousWinGet-Manifest {
             $LastVersion = Split-Path (Split-Path (Get-ChildItem -Path "$AppFolder\..\" -Recurse -Depth 1 -File).FullName ) -Leaf | Sort-Object $ToNatural | Select-Object -Last 1
 
             Write-Host -ForegroundColor 'DarkYellow' -Object "Last Version: $LastVersion"
-            $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$LastVersion"
+            $script:OldManifests = Get-ChildItem -Path "$AppFolder\..\$LastVersion" -Directory
 
             if (-not ($OldManifests.Name -like "$PackageIdentifier*.yaml")) {
```

Comment:
If we use the `-Directory` tag, won't this logic no longer work?

Reply:
@vedantmgoyal2009 I think you need to do something like what was done in Trenly#9 instead of using the `-Directory`

Reply:
Well, I simply copied Easton's suggestion and did not use my brain 🧠